File: public/docs/_examples/pipes/ts/app/hero-list.component.ts
// #docregion
import { Component } from '@angular/core';

import { FetchJsonPipe } from './fetch-json.pipe';

@Component({
  selector: 'hero-list',
  // #docregion template
  template: `
    <h2>Heroes from JSON File</h2>

    <div *ngFor="let hero of ('heroes.json' | fetch) ">
      {{hero.name}}
    </div>

    <p>Heroes as JSON:
    {{'heroes.json' | fetch | json}}
    </p>
  `,
  // #enddocregion template
  pipes: [FetchJsonPipe]
})
export class HeroListComponent { }

File: public/docs/_examples/pipes/dart/lib/hero_list_component.dart
// #docregion
import 'package:angular2/angular2.dart';

import 'fetch_json_pipe.dart';

@Component(
    selector: 'hero-list',
    template: '''
      <h2>Heroes from JSON File</h2>

      <div *ngFor="let hero of ('heroes.json' | fetch) ">
        {{hero['name']}}
      </div>

      <p>Heroes as JSON: {{'heroes.json' | fetch | json}}</p>
    ''',
    pipes: const [FetchJsonPipe])
class HeroListComponent {}
